sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -565,6 +574,11 @@ def pod_name(self) -> str:
             self.pod_name_template, self._pod_counter)
         return f"{self.pod_name_template}-{self._pod_counter}"
 
+    @property
+    def pod_ip_address(self) -> str:
```

Comment:
We don't have something equivalent to this already ?

Didn't spot where it's used...

Reply:
It is used after creation of the DNS pod and for updating DNS config for loader pods in the `_run_stress` method of the YCSB loader class.